From e9cb4d6ad7908282de47e137e92f735e8d7972a9 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.jz@renesas.com>
Date: Thu, 5 Nov 2020 17:08:23 +0700
Subject: [PATCH 452/454] arm64: dts: renesas: hihope-rzg2-ex.dtsi: enable
 canfd channel 0

CANFD channel 0 and CAN0 use same pins, therefore only one can be
enabled at once.

This commit enable CANFD channel 0 for all hihope extension board, thus
have to disable CAN0.

Signed-off-by: Binh Nguyen <binh.nguyen.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi | 16 +++++++++++-----
 1 file changed, 11 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi b/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi
index b89139a..1187ebb 100644
--- a/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi
@@ -31,10 +31,14 @@
 	};
 };
 
-&can0 {
-	pinctrl-0 = <&can0_pins>;
+&canfd {
+	pinctrl-0 = <&canfd_pins>;
 	pinctrl-names = "default";
 	status = "okay";
+
+	channel0 {
+		status = "okay";
+	};
 };
 
 &can1 {
@@ -69,9 +73,11 @@
 		};
 	};
 
-	can0_pins: can0 {
-		groups = "can0_data_a";
-		function = "can0";
+	canfd_pins: canfd {
+		channel0 {
+			groups = "canfd0_data_a";
+			function = "canfd0";
+		};
 	};
 
 	can1_pins: can1 {
-- 
2.7.4

